uint8 MOVE_FORWARD=0
uint8 MOVE_BACKWARD=1
uint8 MOVE_LEFT=2
uint8 MOVE_RIGHT=3
uint8 STOP=4
uint8 PLAY_SONG=5
uint8 STOP_SONG=6

uint8 command
---
bool result
string message
